Xamarin.PropertyEditing.Mac/Controls/CGRectEditorControl.cs

```diff
@@ -8,24 +8,23 @@ internal class CGRectEditorControl
 	{
 		public CGRectEditorControl ()
 		{
-			// TODO localize
 			XLabel.Frame = new CGRect (26, 23, 25, 24);
-			XLabel.StringValue = "X:";
+			XLabel.StringValue = LocalizationResources.XLabel + ":";
```

Comment:
`X` is not translatable in this context. It refers to API. `X, Y, Width, Height` might all be better off not translated as they directly refer to actual C# properties.

Reply:
Is this a major issue if they are already in the resx? Meaning should I pull these out of the resx?

Reply:
You should pull them out of the resx.

Reply:
There is a mechanism for marking words as "do not translate". This is useful for sentences where, for example, we want to put the Product name in there. Or if we refer to API like watchOS "Complications" and don't want that specific word translated, but do want it in the correct part of the translated sentence

Reply:
please pull in this PR if latest changes are ok.